### ISSUE TITLE
VaButton with 2 icons should have same padding as with one icon

### ISSUE THREAD

**Issue by asvae** [epicmaxco/vuestic-ui#3930]
**Vuestic-ui version:** 1.8.0

https://ui-storybook-develop.vuestic.dev/?path=/story/vabutton--default

<img width="458" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/5394573/d9a3b77a-97ec-4eca-9734-61bbcd163cbc">

**Comment by AdamVacha** [comment]
Id like to handle this please. I'm working on it now.

**Comment by AdamVacha** [comment]
So I've submitted my PR but wanted to double check that I understood correctly. I originally wanted to create logic for Vabutton that allows padding in the presence of icon and icon right. However after reviewing documentation I found just changing the preset from plain to primary achieved the same goal and I assume now that is why the issue was marked as super easy. 

<img width="342" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/90470596/2af9fd86-0486-4d9c-9302-cc555ba99016">

Hope this helps.
